pkg/webhooks/resource/handlers.go

```diff
@@ -17,6 +17,7 @@ import (
 	"github.com/kyverno/kyverno/pkg/metrics"
 	"github.com/kyverno/kyverno/pkg/openapi"
 	"github.com/kyverno/kyverno/pkg/policycache"
+	pcex "github.com/kyverno/kyverno/pkg/policyexception"
```

Comment:
`pcex` may be ambiguous to read. Either `exceptions` or `policyexceptions` is clear.

Reply:
👍 Make sense